share/completions/ip.fish

```diff
@@ -194,6 +194,12 @@ function __fish_ip_scope
         host "Address is only valid on this host"
 end
 
+function __fish_list_netnses
```

Comment:
I'd rather use a `__fish_ip` prefix to be consistent with other functions in this file.
Maybe `__fish_ip_netns_list`? I think it's a good idea to use subject-first naming for things in the global namespace.

Reply:
Yeah that seems better for sure. I forgot that these all land in the global namespace